network/identity.go

```diff
@@ -26,35 +26,52 @@ var (
 type identity struct {
 	proto.UnimplementedIdentityServer
 
-	pending     sync.Map
-	pendingSize int64
+	pending sync.Map
+
+	pendingInboundCount int64
+
+	pendingOutboundCount int64
 
 	srv *Server
 
 	initialized uint32
 }
 
-func (i *identity) numPending() int64 {
-	return atomic.LoadInt64(&i.pendingSize)
+func (i *identity) pendingInboundConns() int64 {
+
+	return atomic.LoadInt64(&i.pendingInboundCount)
+}
+
+func (i *identity) pendingOutboundConns() int64 {
+
+	return atomic.LoadInt64(&i.pendingOutboundCount)
 }
 
 func (i *identity) isPending(id peer.ID) bool {
-	val, ok := i.pending.Load(id)
+	_, ok := i.pending.Load(id)
 	if !ok {
 		return false
 	}
-	return val.(bool)
+	return ok
```

Comment:
Why don't you return `ok` directly?

Reply:
I missed it thanks Commit [link](https://github.com/0xPolygon/polygon-sdk/pull/326/commits/01bcc1cb9296c880f2846d10053b176db288f8e4)